Tests/TuistSigningTests/SecurityControllerTests.swift

```diff
@@ -0,0 +1,64 @@
+import TSCBasic
+import Foundation
+import TuistSupport
+import XCTest
+@testable import TuistCoreTesting
+@testable import TuistSigning
+@testable import TuistSupportTesting
+
+final class SecurityControllerTests: TuistUnitTestCase {
```

Comment:
Would it make sense to make this one an integration test with a temporary keychain? That way we'd be able to detect if the contract between the contract and the `security` utility breaks.